programs/cli_wallet/CMakeLists.txt

```diff
@@ -9,20 +9,19 @@ if( GPERFTOOLS_FOUND )
     list( APPEND PLATFORM_SPECIFIC_LIBS tcmalloc )
 endif()
 
-find_package( Editline )
-if ( EDITLINE_FOUND )
-	message ( STATUS "Found Editline; compiling cli_wallet with Editline" )
-	list ( APPEND PLATFORM_SPECIFIC_LIBS editline )
-	add_library (editline STATIC IMPORTED )
-	set_target_properties ( editline PROPERTIES IMPORTED_LOCATION ${Editline_LIBRARY} ) 
-endif()
-
-
-target_link_libraries( cli_wallet
-                       PRIVATE graphene_app graphene_net graphene_chain graphene_egenesis_brief graphene_utilities graphene_wallet fc ${CMAKE_DL_LIBS} ${PLATFORM_SPECIFIC_LIBS} )
-
 if(MSVC)
+  target_link_libraries( cli_wallet
+         PRIVATE graphene_app graphene_net graphene_chain graphene_egenesis_brief graphene_utilities graphene_wallet fc ${CMAKE_DL_LIBS} ${PLATFORM_SPECIFIC_LIBS} )
   set_source_files_properties( main.cpp PROPERTIES COMPILE_FLAGS "/bigobj" )
+else (MSVC)
+   add_library(editline STATIC IMPORTED)
+   set_target_properties(editline PROPERTIES
+         IMPORTED_LOCATION "${CMAKE_SOURCE_DIR}/libraries/fc/vendor/editline/src/project_editline-build/src/.libs/libeditline.a"
```

Comment:
This is not correct.  Lib file should be "BINARY_DIR" based. I think my patch mentioned in a previous comment in this PR would work better (it didn't handle MSVC though).

Reply:
Sorry, I misread your patch several comments above.  It is better than what I've done. I'm stealing it.